scripts/batch_update.py

```diff
@@ -14,11 +14,12 @@
 
 FIELD_NAMES = {'attested': 'nameAttested',
                'language': 'nameLanguage',
+               'romanized': 'nameTransliterated',
+               'transliterated': 'nameTransliterated',
```

Comment:
@paregorios Are you sure you want to map both of these to the `nameTransliterated` field? If both keys are supplied, only one of them will be used.